data/deep missions.txt

```diff
@@ -0,0 +1,1176 @@
+	source 
+		government Republic "Free Worlds" Syndicate Neutral Independent
+	destination Valhalla
+	to offer
+		has "event: deep: keystone research"
+	on offer
+		dialog
+			`You recieve a message from Ivan, the scientists working on the spacial anomaly in Terminus. "Captain, we have finished studying the Quantum Keystones, and they do in fact allow access through the wormhole. Please contact me on <destination> if you wish to continue helping us with our research."`
+
+mission "Deep: Remnant 1A"
+	landing
+	name `Exploring the Unknown`
+	description `Explore the region of space beyond the wormhole in Terminus. Return to <destination> after exploring as much as you can.`
+	source Valhalla
+	waypoint Terminus
+	to offer
+		has "Deep: Remnant: Keystone Research: done"
+		not "knew of stones before scientists"
+	on offer
+		conversation
+			`You contact Ivan after you land and he sends you directions to his research lab, which is located on the outskirts of the spaceport city. When you arrive you are greeted by Ivan, Maria, and a number of other scientists on the team. Ivan begins to inform you on what their research has revealed.`
+			`	"We've analyzed the Quantum Keystones, and surprisingly they do allow entry into the wormhole in Terminus. It would appear that the stones stabilize the wormhole as the ship passes through it, or maybe it is stabilizing the ship in some way and not the wormhole. We'll need much more time to fully understand the true mechanism that is at play here. Whatever the case is, one stone is required for each ship to pass through the wormhole."`
+			`	"We sent a single ship through the wormhole then returned immediately, as we have no idea what dangers may lurk in the region of space that this wormhole leads to. Would you be willing to explore the region for us? You may want to bring a ramscoop, as we are unsure if there will be anywhere to refuel."`
+			choice
+				`	"I'm always willing to go on an adventure."`
+				`	"This sounds too risky for me. You're going to need to find someone else to do the job."`
+					decline
+			
+			`	"Splendid! We will give you a single Quantum Keystone in order to travel through the wormhole. Return here once you have finished exploring."`
+				accept
+	
+	on accept
+		outfit "Quantum Keystone" 1
+	
+	on enter Arculus
+		dialog `This region of space appears to be inhabited. You should attempt to find out more about the beings living here so that you may tell Ivan.`
+	
+	on enter Nenia # Forces the player to check the Ember Waste, but the player could have taken a keystone and seen for themself that they could enter the Ember Waste while the group was doing research, therefore making this mission extraneous. Any possible solutions? Create a C branch for if the player did not know of the Remnant at the time of speaking with Ivan but later learned about them when Ivan was studying the stones?
```

Comment:
There is a bit of similarity with the existing Remnant mission to scan void sprites, so perhaps we condition this mission to not offer if that has been done. If the player goes to explore the Waste while the Deep is researching the stones, they may stay there for some time before returning to the rest of human space (probably will return when the Remnant asks them to fetch Hai stones).
If this mission can't offer, then a C mission would be needed to set the relevant conditions that 1A would otherwise set, and have the player just tell Ivan about the interactions with the Void Sprites